docs/app-guide/add-a-onepassword-secret.rst

```diff
@@ -0,0 +1,116 @@
+###########################################
+Add a secret with 1Password and VaultSecret
+###########################################
+
+Whenever possible, static secrets for applications are stored in a 1Password vault before being automatically synced to the Vault service itself and ultimately to Kubernetes ``Secret`` resources via :ref:`vault-secrets-operator`.
```

Comment:
I think it may be worth clarifying here that it's not really "whenever possible," it's any secret that for one reason or another we can't randomly generate for new installations. So it's any secret for external systems (Slack webhooks, AWS credentials, Google credentials) or that we need to share with other people (Argo CD admin passwords). We prefer to generate random secrets for anything we can (such as in-cluster Redis passwords) and those aren't stored in 1Password.

Reply:
Sounds good. It sounded like we weren't always using 1password and some folks were still using Vault directly, so I wasn't sure on the policy line.

Reply:
Oh, right, the non-us deployments are not using 1Password. My point was more that we don't want to put the ephemeral secrets in 1Password; they should live only in Vault.